example/android/app/src/main/java/com/example/segmentanalyticsreactnative/MainApplication.java

```diff
@@ -13,12 +13,18 @@
 import java.lang.reflect.InvocationTargetException;
 import java.util.List;
 import com.segmentanalyticsreactnative.AnalyticsReactNativePackage;
+import com.segmentanalyticsreactnative.AnalyticsReactNativeModule;
 import com.analyticsreactnativepluginadvertisingid.AnalyticsReactNativePluginAdvertisingIdPackage;
-
+import com.sovranreactnative.Sovran;
+import android.util.Log;
 
 public class MainApplication extends Application implements ReactApplication {
 
+  private AnalyticsReactNativePackage analytics = new AnalyticsReactNativePackage();
+  private Sovran sovran = new Sovran();
```

Comment:
nit: Technically you don't need a reference to this as you are not using any function here directly